admin.adoc

```diff
@@ -547,7 +547,7 @@ You'll need a copy of the GRR source:
 > git clone https://github.com/google/grr.git
 --------------------------------------------------------------------------------
 
-and the latest versions of link:https://www.vagrantup.com/[Vagrant] and VirtualBox installed.
+and the latest versions of link:https://www.vagrantup.com/[Vagrant] and VirtualBox installed. The Linux VM's provided by the project were built with an older version of VirtualBox. It is recommended that you install the plugin link:https://github.com/dotless-de/vagrant-vbguest[vagrant-vbguest] to update the guest additions to the current version your running on the build host.
 
```

Comment:
I was seeing some problems with the centos VM (/grr share not getting mounted) but I've just updated both vagrant and Virtualbox to see if it still exists. It looks like it might be fixed. When did you encounter issues, after kernel upgrade on the guest?

If we make this recommendation we should be setting
```
config.vbguest.no_remote = true
```
Since this is an unverified http download (thanks Oracle):
https://github.com/dotless-de/vagrant-vbguest/blob/06ebf273ea7889b72dfd5b1259abbf795ed849d5/lib/vagrant-vbguest/hosts/virtualbox.rb#L11

But I'm not sure what happens if you have that set but don't have the plugin installed....